src/main/g8/project/plugins.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.5.1")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.5.6")
```

Comment:
A merciful and well needed update at the time of creation but
now superseded. I believe the current template uses 0.5.8.